### PR TITLE
Add support for Listing Group Aliases

### PR DESCRIPTION
**Pull request by adawalli**
Addresses https://github.com/TerryHowe/ansible-modules-hashivault/issues/398

### CODE REVIEW

**Comment by TerryHowe** [comment]
Is it possible to add a test? Will this work on the latest vault container image?

**Comment by adawalli** [comment]
Added a unit test - I tested it using your `start.sh` and `run.sh` scripts, so it does work with latest vault

**Comment by adawalli** [comment]
@TerryHowe - let me know if you needed anything else on this PR. Thanks!

**Comment by adawalli** [comment]
Also, I don't think I can kick off the workflow since I am a first time contributor. Thanks @TerryHowe

**Comment by adawalli** [comment]
Hi @TerryHowe - hope all is well - anything I can do to help get this on main?

File: ansible/modules/hashivault/hashivault_identity_group_alias_list.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+from ansible.module_utils.hashivault import hashivault_argspec
+from ansible.module_utils.hashivault import hashivault_auth_client
+from ansible.module_utils.hashivault import hashivault_init
+from ansible.module_utils.hashivault import hashiwrapper
+
+ANSIBLE_METADATA = {'status': ['stableinterface'], 'supported_by': 'community', 'version': '1.1'}
+DOCUMENTATION = '''
+---
+module: hashivault_identity_group_alias_list
+version_added: "4.6.9"
```
Comment (TerryHowe) [inline ansible/modules/hashivault/hashivault_identity_group_alias_list.py:11]:
```suggestion
version_added: "4.7.1"
```

**Comment by adawalli** [comment]
Anything else needed from me here?

**Comment by TerryHowe** [comment]
Not sure why I can't rerun the lint, something si wrong

**Pull request merged**